frontend/lib/ui/Form/Form.tsx

```diff
@@ -101,13 +89,12 @@ export const Form = React.forwardRef<HTMLFormElement, FormProps>(({ children, sk
   React.useEffect(() => {
     const inputs = innerRef.current?.querySelectorAll("input, select, textarea") as NodeListOf<HTMLInputElement>;
     inputList.current = Array.from(inputs);
-
-    //filter out inputs we should skip
-    if (skip && skip.length) {
-      inputList.current = inputList.current.filter((input) => !skip.includes(input.id));
-    }
     submitButton.current = innerRef.current?.querySelector("button[type=submit]") as HTMLButtonElement | null;
-  }, [children, skip]);
+  }, [children]);
+
+  React.useEffect(() => {
+    document.getElementById("form-title")?.focus();
```

Comment:
instead of id, maybe find the first header element?
or find by className

Reply:
Changed in d1bf0ed431654f0ec14291f58a46a924fc3242c0